CONTRIBUTING.md

```diff
@@ -1,28 +1,35 @@
- # Contributing to Mongock project
+# Contributing to Mongock project
 
 :tada: First off, thanks for taking the time to contribute! :tada:
 
  The following is a set of guidelines for contributing to Mongock project which is hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel
  free to propose changes to this document in a pull request.
 
 
-## Getting started
+## Getting started :rocket:
 
-> :boom::rocket: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out the best candidates and which of them are rewarded!!
+> :boom: Visit our page of [requested contribution](https://github.com/mongock/mongock/labels/contribution-requested) to find out the best candidates and which of them are rewarded!!
 
-Contributions are made to this repo via **issues**, **discussions** and **pull request**.
+Contributions are made to this repo via **issues**, **discussions** and **code contributions**.
 
-Within the Pull Request section, there is a special kind of contribution that deserves special attention:  [Creation of new drivers](#providing-a-new-driver)
-
-
-## Issues
+## Issues :bug:
 For raising a new issue, please visit this [link](https://github.com/mongock/mongock/issues/new?assignees=&labels=&template=bug_report.md&title=), fills the template and submit it. 
 You will hear from us shortly.
 
-## Discussions
-For general **Q&A**s, **Ideas**, **comment on any release** or anything you want to raise, but it's not an issue, please visit the [discussion page](https://github.com/mongock/mongock/discussions)
+**Contributor raising a valid issue(or solving it) will get listed in our [contributor list](https://github.com/mongock/mongock/blob/master/CONTRIBUTORS.md).**
+
+## Discussions :loud_sound:
+For general **Q&A**s, **ideas**, comment on any **release** or anything you want to raise, but it's not an issue, please visit the [discussion page](https://github.com/mongock/mongock/discussions).
+
+
+**Contributor starting a valid discussion(or participates in it) will get listed in our [contributor list](https://github.com/mongock/mongock/blob/master/CONTRIBUTORS.md).**
```

Comment:
Agree, we could and should mention anyone who is helping. But would we want to differenciate in what someone is helping on given the potential effort?

Reply:
Take a look to the `code contribution` section, it says , they will be listed with special mention

Is that what you mean?